piqueserver/core_commands/server.py

```diff
@@ -1,4 +1,15 @@
+import random
+
+from twisted.logger import Logger
+
 from piqueserver.commands import command, join_arguments
+from piqueserver.config import config
+
+log = Logger()
+
+name_option = config.option(
```

Comment:
note: this is declared but not used in this file

Reply:
Oh, this was a remnant from before I added set_server_name